OPAL/bc/src/main/scala/org/opalj/bc/Assembler.scala

```diff
@@ -752,6 +754,9 @@ object Assembler {
             writeShort(methods.size)
             methods foreach { m ⇒
                 serialize(m)
+                if  ((ACC_STRICT.mask & m.access_flags) != 0 && (classFile.major_version < 46) || (classFile.major_version > 60)) {
```

Comment:
There's an issue here with the parentheses. Unless I'm mistaken, this will always print an error for classfile version > 60 because the && binds stronger than the ||. I guess the parentheses right before and after the || should be removed to fix this?